ReactWindows/ReactNative.ChakraBridge/JsrtJavaScriptExecutor.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "pch.h"
+#include <jsrt.h>
+#include <ppltasks.h>
+#include "ChakraHost.h"
+#include "ChakraStringResult.h"
+
+using namespace concurrency;
+using namespace Platform;
+using namespace Windows::Foundation;
+using namespace Windows::Storage;
+using namespace Windows::Storage::Streams;
+
+namespace ReactNative { namespace ChakraBridge {
+
+[Windows::Foundation::Metadata::WebHostHidden]
```

Comment:
@rozele can be removed now that we're not using `Windows::Data::Json`.  It was to preclude usage from the JS runtime because it was using XAML data structures